crates/shared/src/http_solver_api/model.rs

```diff
@@ -132,6 +132,15 @@ pub struct FeeModel {
     pub token: H160,
 }
 
+#[serde_as]
+#[derive(Clone, Debug, Deserialize, PartialEq)]
+pub struct InteractionData {
+    pub target: H160,
+    pub value: U256,
+    pub call_data: Bytes<Vec<u8>>,
```

Comment:
The Bytes type is not right. This more of an implementation detail in ethcontract but here it matters for serialization. It can be plain Vec<u8> in which case the it would be formatted as a json list like [1,255,4,3].